varnish-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,137 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'varnish-cache',
+        rules: [
+          {
+            alert: 'VarnishCacheLowCacheHitRate',
+            expr: |||
+              increase(varnish_main_cache_hit[10m]) / (clamp_min((increase(varnish_main_cache_hit[10m]) + increase(varnish_main_cache_miss[10m])), 1)) * 100 < %(alertsWarningCacheHitRate)s and (increase(varnish_main_cache_hit[10m]) + increase(varnish_main_cache_miss[10m]) > 0)
```

Comment:
@rgeyer I'm not sure, but do you think there's any way we could optimise this?
A little worried about the interval timing here

Reply:
Unfortunately, for alerts we don't have facilities like `$__rate_interval` or `$__range`, so a static interval is necessary. I'd argue this is actually "better" than most mixins which use 5min as the interval and *could* lack sufficient data with the 1m scrape interval standard for Grafana Cloud.

So, this is as good as it gets.

Reply:
Ahh good point, I'd forgotten they weren't available for alert queries! Thanks for double checking :D